CONTRIBUTING.md

```diff
@@ -0,0 +1,143 @@
+# Table of Contents
+1. [Code of Conduct](#codeOfConduct)
+2. [How to Contribute](#howToContribute)
+    1. [Reporting Bugs / Suggesting Enhancements](#bugAndEnhancements)
+    2. [Submitting a Pull Request](#pullRequest)
+3. [Development Setup](#developmentSetup)
+4. [Branching & Naming Conventions](#branchesAndNames)
+5. [Commit Message Guidelines](#commits)
+6. [Style & Linting](#styleAndLinting)
+7. [Testing](#testing)
+8. [Documentation](#documentation)
+9. [Release Process (maintainers only)](#release)
+10. [License](#license)
+
+## Code of Conduct <a name="codeOfConduct"></a> 
+
+We adopt the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/). By participating in this project you agree to abide by its terms. Please read it before interacting with the community.
+
+## How to Contribute <a name="howToContribute"></a>
+
+### Reporting Bugs / Suggesting Enhancements <a name="bugAndEnhancements"></a>
```

Comment:
This reads like how to contribute when targeting code -- we don't really have any code.

Perhaps a better section would be how to request a new experimental technique, how to enhance an existing term, how to report a concern/problem, ...